OrbitQt/CMakeLists.txt

```diff
@@ -61,7 +61,8 @@ target_sources(
           processlauncherwidget.cpp
           servicedeploymanager.cpp
           TopDownViewItemModel.cpp
-          topdownwidget.cpp)
+          topdownwidget.cpp
+		  ../icons/orbiticons.qrc)
```

Comment:
It seems some tabs slipped into the file.

Reply:
Fixed